### ISSUE TITLE
[new site support] bavarikon.de again

### ISSUE THREAD

**Issue by frederic2de** [lovasoa/dezoomify#472]
### Site name and desciption
Hi, the solution in issue #298 seems not to work any more.

### Example URLs
I would like to download the image on https://www.bavarikon.de/object/bav:BSB-BAR-0000000000160585

So I tried https://api.digitale-sammlungen.de/iiif/image/v2/bav:BSB-BAR-0000000000160585/info.json with dezoomify (and dezoomify-rs) with and without the recommended proxy in the console in FF and Chrome.

### Current error message
Due to a 403 forbidden error the first (and all other tiles) can't be downloaded.

Thanks in advance and my deepest respect for your great work!

**Comment by lovasoa** [comment]
You can have a look at this wiki page : https://github.com/lovasoa/dezoomify/wiki/How-to-access-protected-pages

In your case, the dezoomify-rs command is 

```
./dezoomify-rs --header "Referer: https://www.bavarikon.de/" "https://api.digitale-sammlungen.de/iiif/image/v2/bav:BSB-BAR-0000000000160585/info.json" 
```

However, this currently does not work because dezoomify-rs overrides the Referer header when it receives a redirection from the server. This is a one-line change, I'll let you know when this is fixed.

**Comment by lovasoa** [comment]
This is fixed in dezoomify-rs v2.4.1

![dezoomified_10](https://user-images.githubusercontent.com/552629/94935306-317cc780-04cd-11eb-9d64-70528c39b35b.jpg)
